examples/README.md

```diff
@@ -0,0 +1,6 @@
+# `react-plotly.js-editor` examples
+
+* [Simple `react-plotly.js-editor` example](simple): `DefaultEditor`, synchronous data, top-level component state management
+* [Async-data `react-plotly.js-editor` example](async-data): `DefaultEditor`, asynchronous data, top-level component state management
```

Comment:
fwiw now that we don't need an evented system I think we should change this example to use `fetch` and `promises` instead of events. That will be useful to a wider percentage of users. Not necessary in this PR.

Reply:
Agreed.